tests/ert_tests/shared/test_log_process_usage.py

```diff
@@ -0,0 +1,26 @@
+import logging
+from ert_shared.main import log_process_usage
+
+
+def test_valid(caplog):
+    caplog.set_level(logging.INFO)
+    log_process_usage()
+
+    assert len(caplog.records) == 1
+
+
+def test_unsupported_platform(caplog, monkeypatch):
+    """
+    This test supposes that some exception occurs due to the current system not
+    supporting the 'resource' package. The correct behaviour is to silently
+    ignore errors because this is only meant for logging.
+    """
+    import resource
+
+    monkeypatch.delattr(resource, "getrusage")
+    caplog.set_level(logging.INFO)
```

Comment:
Personally I prefer contextmanager, but up to you:
```python
with caplog.at_level(logging.INFO):
```

Reply:
Done.